src/ImageRunModal.jsx

```diff
@@ -934,15 +970,18 @@
                                  formclass='publish-port-form'
                                  label={_("Port mapping")}
                                  actionLabel={_("Add port mapping")}
+                                 validationFailed={dialogValues.validationFailed.publish}
+                                 onValidationChange={value => this.dynamicListOnValidationChange(value, "publish")}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1354-20230907-093411-a9f37bd4-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)